_about/groups/task-forces/coga.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Cognitive and Learning Disabilities Accessibility Task Force (COGA)
+ref: /about/groups/task-forces/coga/
```

Comment:
```suggestion
ref: /about/groups/task-forces/coga/
permalink: /about/groups/task-forces/coga/
```
@iadawn This page is missing a `permalink` attribute in the frontmatter, causing the build to fail.